python/ray/node.py

```diff
@@ -108,7 +145,87 @@ def raylet_socket_name(self):
         """Get the node's raylet socket name."""
         return self._raylet_socket_name
 
-    def prepare_socket_file(self, socket_path):
+    def create_redis_client(self):
+        """Create a redis client."""
+        return ray.services.create_redis_client(
+            self._redis_address, self._ray_params.redis_password)
+
+    def get_temp_dir_path(self):
+        """Get the path of the temporary directory."""
+        return self._temp_dir
+
+    def get_logs_dir_path(self):
+        """Get the path of the log files directory."""
+        return self._logs_dir
+
+    def get_sockets_dir_path(self):
```

Comment:
All of these methods you added are internal helper methods, right? Please prefix them with `_`

Reply:
No, they are meant to be public methods.